PowerShell.Common.props

```diff
@@ -137,9 +137,8 @@
 
     <TargetFramework>net7.0</TargetFramework>
     <LangVersion>10.0</LangVersion>
-    <!-- disable ReadyToRun due to issue: https://github.com/dotnet/runtime/issues/65412
```

Comment:
Please update this with the new issue preventing us from using ReadyToRun.

https://github.com/dotnet/runtime/issues/72550